### DISCUSSION TITLE
Info versione nuova -si prega di rispondere.

### DISCUSSION THREAD

**Discussion by gianlucaaiello56** [iisgiua/giuaschool#607] [Richieste di aiuto] [answered]
@trinko il registro chiede l'aggiornamento
"Sono presenti modifiche successive al rilascio della versione 1.5.6" che da quanto ho capito è stato rilasciato 19h fa.
E' sicuro? Posso procedere ad installarlo ?

**Comment by trinko** [comment, marked as answer]
Si, sono delle correzioni varie, meglio installarle.
